tests/spread/witchcraft/pack-eol/task.yaml

```diff
@@ -1,42 +1,56 @@
 summary: test witchcraft packing on an EOL base.
 
-systems:
-  - -ubuntu-20.04-64 # Overlay step freezes forever.
-
 prepare: |
-  sudo apt-get --yes install faketime
+  timedatectl set-ntp false
+  date +%s > .real-time-now
+
+  # Change to the current base
+  . /etc/os-release
+  sed -i "s/base: REPLACE_THIS/base: ubuntu@${VERSION_ID}/" witchcraft.yaml
```

Comment:
This is a duplicate of the 3 following lines (I suspect an error when rebasing).

Reply:
Yep, it is. Thanks!

```suggestion
```